modules/oas-page-builder/tests/unit/services/pageBuilder.test.ts

```diff
@@ -14,10 +15,17 @@ jest.mock('../../../src/services/redocExecutor', () => ({
   })),
 }));
 
+// const mockSaveSuccessfulBuildVersionData = jest.fn();
```

Comment:
(Nit) Is this something we should delete?

Reply:
Thank you!